src/Microsoft.ML.Data/Transforms/TermTransformImpl.cs

```diff
@@ -1051,7 +1051,6 @@ public override void AddMetadata(ColumnMetadataInfo colMetaInfo)
                         MetadataUtils.MetadataGetter<VBuffer<DvText>> getter =
                             (int iinfo, ref VBuffer<DvText> dst) =>
                             {
-                                _host.Assert(iinfo == _iinfo);
```

Comment:
> _host.Assert(iinfo == _iinfo); [](start = 31, length = 31)

Now this is a funny instance. 
In fact, this getter incorrectly assumes that it will be always called with the same `iinfo` as it was created for. In reality, this getter is invoked by the `MetadataRow` as a *value* getter (not metadata getter), and the index here is the column index in the metadata row.

I assume this confusion will go away eventually, because there won't be 'metadata getters', but we can expect such asserts to fire elsewhere, I guess.